src/Utilities/useConnectionStatus.js

```diff
@@ -22,7 +24,10 @@ export const useConnectionStatus = (remediation) => {
       }
     };
 
-    fetchData();
+    mounted.current && fetchData();
```

Comment:
It's rather the `setisConnected` on line 19 that would need to be prevented in case the component is unmounted while the fetch is happening.

Reply:
ahh I see. Changes are up